directory/src/main/java/org/ecclesia/directory/domain/Organization.java

```diff
@@ -3,14 +3,16 @@
 import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import java.io.Serializable;
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 /**
  * Represents an organization. Data transfer object.
  */
 @Entity
 @Table(name = "organization")
-@Data
+@Getter @Setter @NoArgsConstructor
```

Comment:
**Reporter**: Checkstyle
**Rule**: AnnotationLocationMostCases
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-7/directory/src/main/java/org/ecclesia/directory/domain/Organization.java L15

Annotation &#39;NoArgsConstructor&#39; should be alone on line.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1698978768>*